src/main/java/org/jabref/logic/importer/Importer.java

```diff
@@ -97,6 +108,24 @@ public ParserResult importDatabase(Path filePath, Charset encoding) throws IOExc
         }
     }
 
+    protected static Charset getCharset(BufferedInputStream bufferedInputStream) {
+        Charset charset = StandardCharsets.UTF_8;
```

Comment:
Shouldn't this fallback to the System default charset maybe?

Reply:
Locally failed tests went up from 273 to 288 🙈

Reply:
One of the problem I now encounter as well ( and I wonder why the tests previously didn't fail, is that the bibtex string resolving on import is making problems)